eng/pipelines/coreclr/crossgen2.yml

```diff
@@ -57,6 +57,23 @@ extends:
             displayNameArgs: R2R_CG2
             liveLibrariesBuildConfig: Release
 
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          buildConfig: Release
+          platforms:
+            - linux_x64
+            - osx_x64
+            - windows_x64
+          jobParameters:
+            testGroup: innerloop
+            buildArgs: -s clr+libs+libs.tests
+                       -c $(_BuildConfig) -test
```

Comment:
This will run the test on the build machine. It is not how we run the tests. The tests should be sent to Helix to run instead.

It is also causing https://github.com/dotnet/runtime/issues/94190. Build machines do not have all dependencies required by the tests.